dockerfile-mode.el

```diff
@@ -63,14 +63,17 @@ Each element of the list will be passed as a separate
   '((t (:inherit (font-lock-constant-face bold))))
   "Face to highlight the base image alias inf FROM ... AS <alias> construct.")
 
+(defconst dockerfile--from-regex
```

Comment:
Is there a reason you use double dashes rather than single dash like the rest of the code?

Reply:
In Elisp the general convention is that a single dash exposes a public API and double dash marks a private API or variables.  Since this regexp can possibly change in the future I don't think it's advisable to "expose" it as public constant.

https://github.com/bbatsov/emacs-lisp-style-guide#naming